Gemfile

```diff
@@ -25,6 +25,7 @@ manageiq_plugin "manageiq-providers-azure"
 manageiq_plugin "manageiq-providers-hawkular"
 manageiq_plugin "manageiq-providers-kubernetes"
 manageiq_plugin "manageiq-providers-lenovo"
+manageiq_plugin "manageiq-providers-microsoft"
```

Comment:
same here. point to your repo to see if travis turns green.
and put the Gemfile change into its own commit. this way you can run the extraction again and again.